frontend/src/concepts/pipelines/content/createRun/contentSections/RunTypeSection.tsx

```diff
@@ -0,0 +1,69 @@
+import React from 'react';
+
+import { Alert, AlertActionCloseButton, Button, FormSection } from '@patternfly/react-core';
+import { useNavigate, useParams } from 'react-router-dom';
+
+import { PipelineRunTabTitle, PipelineRunType } from '~/pages/pipelines/global/runs';
+import {
+  CreateRunPageSections,
+  runPageSectionTitles,
+} from '~/concepts/pipelines/content/createRun/const';
+import { PipelineRunSearchParam } from '~/concepts/pipelines/content/types';
+
+interface RunTypeSectionProps {
+  runType: PipelineRunType;
+}
+
+export const RunTypeSection: React.FC<RunTypeSectionProps> = ({ runType }) => {
+  const navigate = useNavigate();
+  const { namespace } = useParams();
+  const [isAlertOpen, setIsAlertOpen] = React.useState(true);
+
+  let runTypeValue = 'Run once immediately after creation';
+  let alertProps = {
+    title: 'Go to Schedules to create schedules that execute recurring runs',
+    label: `Go to ${PipelineRunTabTitle.Schedules}`,
+    navSearch: `?${PipelineRunSearchParam.RunType}=${PipelineRunType.Scheduled}`,
+  };
+
+  if (runType === PipelineRunType.Scheduled) {
+    runTypeValue = 'Schedule recurring run';
+    alertProps = {
+      title: 'Go to Active runs to create a run that executes once immediately after creation.',
+      label: `Go to ${PipelineRunTabTitle.Active}`,
+      navSearch: `?${PipelineRunSearchParam.RunType}=${PipelineRunType.Active}`,
+    };
+  }
+
+  return (
+    <FormSection
+      id={CreateRunPageSections.RUN_TYPE}
+      title={runPageSectionTitles[CreateRunPageSections.RUN_TYPE]}
+    >
+      {runTypeValue}
+
+      {isAlertOpen && (
+        <Alert
+          isInline
+          variant="info"
+          title={alertProps.title}
+          actionLinks={
+            <Button
+              isInline
+              variant="link"
+              onClick={() =>
+                navigate({
+                  pathname: `/pipelineRuns/${namespace}`,
```

Comment:
use the helper functions that juntao implemented
https://github.com/opendatahub-io/odh-dashboard/blob/main/frontend/src/routes/pipelines/global.ts#L34C14-L34C40

Make sure you do this for all the redirects

Reply:
Yeah, these didn't exist when I opened this PR, so I'll update with them.

Reply:
thanks